crowdsourcing/viewsets/task.py

```diff
@@ -1,14 +1,42 @@
-__author__ = 'elsabakiu, dmorina, asmita, megha'
+__author__ = 'elsabakiu, dmorina, asmita, megha,kajal'
 
 from crowdsourcing.serializers.task import *
-from rest_framework import viewsets
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+from rest_framework.decorators import detail_route, list_route
+from crowdsourcing.models import Worker,TaskWorker,Module
+from rest_framework import mixins
+from django.shortcuts import get_object_or_404
 
 class TaskViewSet(viewsets.ModelViewSet):
-    from crowdsourcing.models import Task
-    queryset = Task.objects.all()
-    serializer_class = TaskSerializer
+	from crowdsourcing.models import Task
+	queryset = Task.objects.all()
+	serializer_class = TaskSerializer
 
+	@detail_route(methods=['post'])
```

Comment:
so can i give this permissions:
@detail_route(methods=['post'],permission_classes=[IsProjectCollaborator,])
so that they can have permission to collaborate on it and run it?